mycore-base/src/main/java/org/mycore/tools/MyCoReWebPageProvider.java

```diff
@@ -149,11 +149,14 @@ public Element addSection(String title, List<Content> contentList, String lang)
         if (lang != null) {
             section.setAttribute(XML_LANG, lang, Namespace.XML_NAMESPACE);
         }
-        if (title != null && !title.equals("")) {
+        if (title != null && !title.isEmpty()) {
             section.setAttribute(XML_TITLE, title);
         }
         for(Content content : contentList) {
-            if(content instanceof Text) {
+            if(content instanceof Text text) {
+                if(text.getTextTrim().isEmpty()) {
```

Comment:
suggestion: could use `text.isBlank()` instead; no need to create the trimmed sting